source/clear-linux/guides/network/network-bonding.rst

```diff
@@ -86,25 +91,25 @@ __ https://www.freedesktop.org/software/systemd/man/systemd.netdev.html#id-1.20.
       [Link]
       MTUBytes=9000
 
-   Since ``bond1`` is a virtual interface, it has no concept of physical link
-   status. The ``BindCarrier`` directive indicates that the link status of this
-   interface is determined by the status of the listed slave devices.
+   `bond1` is a virtual interface with no physical link status.
 
-   This is the logical interface, so assign it an IP address. DHCP is more
-   complicated with bonded interfaces, and is not covered in this example.
+   `BindCarrier` indicates that the `bond1` link status is determined by the
+   status of the listed slave devices.
 
-   This file also enables jumbo frames of up to 9000 bytes. This option must be
-   enabled for all slave interfaces *and* the bonded interface, in order to take
-   effect.
+   `Address` contains an IP address that you assign to the logical interface.
+   DHCP bonded interfaces are complex and outside the scope of this example.
 
-5. Apply the new network configuration:
+   `MTUBytes` must be the same on all slave interfaces and on the bonded
+   interface for successful operation.
 
-   .. code-block:: console
+#. Apply the new network configuration with the command:
+
+   .. code-block:: bash
 
-      # systemctl restart systemd-networkd
+      sudo systemctl restart systemd-networkd
 
-   The MTU settings will not take effect until a reboot, or if you explicitly
-   apply them via ``ifconfig``, for example.
+   The `MTUBytes` settings do not take effect until you reboot or you explicitly
```

Comment:
"manually" is probably better than "explicitly"

Reply:
Fixed.